setup.py

```diff
@@ -22,224 +22,19 @@
 from pathlib import Path
 
 from setuptools import find_packages, setup, Command
```

Comment:
Can we break out this into a separate PR? I'd prefer to merge the pyproject changes in first and then rip out setup, yeah?

Reply:
What are you referring to specifically? the PR moves stuff from one to the other, you can't really have the same things configured in both files at the same time.

Reply:
🤦  yeah nvm